Employees/Entities/EmployeeType.cs

```diff
@@ -4,10 +4,10 @@
 
 namespace Employees.Entites
 {
-   public enum EmployeeType
+    public enum EmployeeType
     {
         contract,
         temp,
-        Permanent,   
+        Permanent,
     }
 }
```

Comment:
You could change the casing of the first two options here. It'll work just fine as it is, but the convention for enum values is to use PascalCase.

Reply:
I will do that thank you.